docs/Documentation/Applications/avizo.md

```diff
@@ -0,0 +1,30 @@
+#Using Avizo Software on the Eagle System
+Learn about using Avizo 3D analysis software for scientific and industrial data on the Eagle System.
+
+##Introduction
+Avizo software is a powerful, multifaceted commercial software tool for visualizing, manipulating, and understanding scientific and industrial data. 
+NREL licenses a limited number of Avizo tokens, allowing a small number of Avizo instances to be run lab-wide.
+
+##Running Remotely
+Avizo is installed and can be run remotely from the Eagle visualization node. 
+First, launch a TurboVNC remote desktop. 
+Then from a terminal in that remote desktop:
+
+``` bash
+% module load avizo 
+% vglrun avizo
+```
+
+##Running Locally
+Avizo can also be run on a local desktop connected to the NREL network — the machine must be connected to the network to access the license server.
+
+##Install the Software
+First install the software:
```

Comment:
These are all rendering on the same line. Should go on individual line for each.